test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -643,6 +643,25 @@ add_specs suite_builder =
             Decimal.new "10" . pow -1 . should_fail_with Arithmetic_Error
             Decimal.new "10" . pow 99999999999 . should_fail_with Arithmetic_Error
 
+    suite_builder.group "signs" group_builder->
+        group_builder.specify "should calculate abs correctly" <|
+            Decimal.new "12.345" . abs . should_equal 12.345
+            Decimal.new "-12.345" . abs . should_equal 12.345
+            Decimal.new "12.345E97" . abs . should_equal (Decimal.new "12.345E97")
+            Decimal.new "-12.345E97" . abs . should_equal (Decimal.new "12.345E97")
+
+        group_builder.specify "should calculate negate correctly" <|
+            Decimal.new "12.345" . negate . should_equal -12.345
+            Decimal.new "-12.345" . negate . should_equal 12.345
+            Decimal.new "12.345E97" . negate . should_equal (Decimal.new "-12.345E97")
+            Decimal.new "-12.345E97" . negate . should_equal (Decimal.new "12.345E97")
+
+        group_builder.specify "should calculate signum correctly" <|
+            Decimal.new "12.345" . signum . should_equal 1
+            Decimal.new "-12.345" . signum . should_equal -1
+            Decimal.new "12.345E97" . signum . should_equal 1
+            Decimal.new "-12.345E97" . signum . should_equal -1
```

Comment:
this effectively boils down to unit testing the Java implementation, but just for completeness I'd check also the case of `Decimal.new "0"` because it is an edge case different than the ones shown here.

Reply:
Done